execution/engine/engine.go

```diff
@@ -130,11 +130,11 @@ func (ee *Engine) NotifyForkchoiceUpdate(
 	return payloadID, latestValidHash, nil
 }
 
-// VerifyAndNotifyNewPayload verifies the new payload and notifies the
+// NotifyNewPayload notifies the execution client of the new payload
 // execution client.
```

Comment:
ah sorry my suggestion was wrong, we need to cleanup this as well
```suggestion
// NotifyNewPayload notifies the execution client of the new payload.
```